tvb_epilepsy/base/disease_hypothesis.py

```diff
@@ -4,75 +4,231 @@
 """
 from collections import OrderedDict
 
-import numpy
+import numpy as np
 
-from tvb_epilepsy.base.h5_model import prepare_for_h5
-from tvb_epilepsy.base.utils import formal_repr
+from tvb_epilepsy.base.utils import initialize_logger, formal_repr, ensure_list, dicts_of_lists_to_lists_of_dicts, \
+                                    linear_index_to_coordinate_tuples
+from tvb_epilepsy.base.h5_model import convert_to_h5_model
+
+from tvb_epilepsy.base.model_configuration import ModelConfiguration
```

Comment:
This import should not be here! 
It ruins the architectural design of the package (a ModelConfiguration is built after and from a Hypothesis, we should not require it back here)